docs-public/snippets.js

```diff
@@ -110,23 +111,24 @@ var plunkerTemplate = kendo.template(
         .event-log li:last-child { margin-bottom: -1px;}\
     </style>\
     <script src="https://unpkg.com/core-js/client/shim.min.js"></script>\
+    # if (data.platform === \'builder\') { # <script src="https://unpkg.com/zone.js"></script> #}#\
     #= data.cdnResources #\
     <script src="https://unpkg.com/systemjs@0.19.27/dist/system.js"></script>\
     <script src="#: data.exampleRunner #"></script>\
     <script>\
         var runner = new ExampleRunner("#= data.platform #");\
         runner.configure(System, { npmUrl: "#: data.npmUrl #", modules: ' + JSON.stringify(moduleDirectives) + ', language: "#: data.language #", trackjs: #= data.track # });\
         # for (var i = 0; i < data.files.length; i++) { #\
-        runner.register("#= data.files[i].name #", "#= data.files[i].content #");\
+        runner.register("#= data.root #", "#= data.files[i].name #", "#= data.files[i].content #");\
         # } #\
-        runner.start(System);\
+        runner.start(System, "#= data.importRoot || \'app\' #");\
     </script>\
 </head>\
 <body>\
     #= data.html #\
-    <my-app>\
+    <app-root>\
```

Comment:
I'm afraid that this change will require all Angular demos to change, as they currently [reference the `my-app` container](https://github.com/telerik/kendo-angular-grid/search?utf8=%E2%9C%93&q=my-app&type=). We could apply the change across all repos, but I'm not sure if that is necessary.

Reply:
Ops, that is a mistake, will change this to be compliant with the rest of the demos :)

Reply:
Done